.github/dependabot.yml

```diff
@@ -1,34 +0,0 @@
-# Please see the documentation for all configuration options:
-# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
```

Comment:
I believe that this is required for the open source compliance check.
But it does not validate the contents so you can change the intervals / directories now that renovate is being used.